src/MongoDB.Analyzer/Core/Poco/PocoAnalyzer.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright 2021-present MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using MongoDB.Analyzer.Core.Utilities;
+
+namespace MongoDB.Analyzer.Core.Poco;
+
+internal static class PocoAnalyzer
+{
+    public static bool AnalyzePoco(MongoAnalysisContext context)
+    {
+        var sw = Stopwatch.StartNew();
+        var stats = AnalysisStats.Empty;
+        ExpressionsAnalysis pocoAnalysis = null;
+
+        try
+        {
+            context.Logger.Log("Started JSON analysis");
+
+            pocoAnalysis = PocoExpressionProcessor.ProcessSemanticModel(context);
+            stats = ReportJsonOrInvalidExpressions(context, pocoAnalysis);
+
+            sw.Stop();
+            context.Logger.Log($"JSON analysis ended: with {stats.JsonCount} JSON translations, {stats.DriverExceptionsCount} unsupported expressions, {stats.InternalExceptionsCount} internal exceptions in {sw.ElapsedMilliseconds}.");
```

Comment:
as `ms` to `{sw.ElapsedMilliseconds}` everywhere (missed that in other analyzers).

Reply:
Done